### DISCUSSION TITLE
grocy support for online ordering?

### DISCUSSION THREAD

**Discussion by 2t2operator** [hassio-addons/addon-grocy#139] [Ideas]
Hey Frenck any thoughts on adding support for instacart or something similar? 

Some mobile apps use what appears to be an iframe, the shopping list "item" is forwarded to the iframe to allow the user to select the product based on choices available from the store.

Once the item is selected in the iframe it is added to the Cart (Grocery Store).  Process repeats until all items on the shopping list are handled, e.g., skip it, add it, etc.  This is all handled on the Grocery Store side, we just need the lookup or push to take place :-) 

Example:
grocy shopping list
Item: Ground Beef

LOOKUP: https://www.instacart.com/store/publix/search_v3/ground%20beef
NOTE: Not sure what other parameters on the URL are required if any?

Displays all of the available choices of Ground Beef for my example at Publix
Select Product > Add to Cart
Next item

Once shopping is complete, grocy inventory is updated! This should reduce the amount of "scanner time". On that note, now just have to figure out how to get the data from the online service; other then a image!

**Comment by jackwilsdon** [comment]
This sounds like a feature request for grocy itself, not the grocy home assistant addon: https://github.com/grocy/grocy

**Reply by frenck** [reply]
Correct, @2t2operator Please suggest this upstream

**Reply by 2t2operator** [reply]
Thanks all, being new to Github, I am not able to find how to move this upstream?  I can see "Choose another repository Home Assistant Community Add-ons owns to move this discussion to".  Is there a way to simply close this out here ?

**Reply by frenck** [reply]
You can't move, you'd. have to. re-created.

Upstream: https://github.com/grocy/grocy

There is no close method in discussions.
